.buildkite/pipeline.smoke.yml

```diff
@@ -0,0 +1,239 @@
+steps:
+  - label: ':docker: Build Android maze runner image'
+    key: "maze-runner"
+    timeout_in_minutes: 20
+    plugins:
+      - docker-compose#v3.3.0:
+          build: android-maze-runner
+          args:
+            - TESTS_DIR=features/smoke_tests
+
+  - label: ':android: NDK 16b SDK 4.4 Instrumentation tests'
```

Comment:
Running the instrumentation tests on just 16b + 4.4 would be fine for a quick CI run - there's no advantage from my perspective in running these tests 4x.